Client/src/main/java/io/github/jwdeveloper/tiktok/TikTokLiveHttpClient.java

```diff
@@ -210,49 +188,29 @@ public LiveConnectionData.Response fetchLiveConnectionData(LiveConnectionData.Re
         }
     }
 
-    SignServerResponse getSignedUrl(String roomId) {
-        var urlToSign = httpFactory
-                .client(TikTokLiveHttpClient.TIKTOK_URL_WEBCAST + "im/fetch")
-                .withParam("room_id", roomId)
-                .build()
-                .toUrl();
+    HttpResponse<byte[]> getStarterPayload(String room_id) {
+        HttpClientBuilder builder = httpFactory.client(TIKTOK_SIGN_API)
+            .withParam("client", "ttlive-java")
```

Comment:
I think we should keep client parameter for metricts. Or it was removed from sign server?

Reply:
Correct, we no longer need the url. Only the current parameters and room_id passing directly to the signing server.